modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceCompare.testcase

```diff
@@ -37,12 +37,37 @@ definition {
 	}
 
 	@description = "COMMERCE-5911. As a buyer, I want to select items in Search Results page for compare and see those items in MiniCompare widget at the bottom pf the page"
-	@ignore = "true"
 	@priority = "3"
 	test AssertComparisonItemsAreVisibleOnMiniCompare {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-5911 AssertComparisonItemsAreVisibleOnMiniCompare pending implementation
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Account Management");
+
+		CommerceAcceleratorsInitializer.createNewAccountViaFront(
+			accountName = "Commerce Account",
+			adminEmailAddress = "test@liferay.com");
+
+		CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Catalog");
+
+		Check.checkToggleSwitch(
+			key_productName = "U-Joint",
+			locator1 = "CommerceEntry#SELECT_CHECKBOX");
+
+		Check.checkToggleSwitch(
+			key_productName = "Transmission Cooler Line Assembly",
+			locator1 = "CommerceEntry#SELECT_CHECKBOX");
+
+		Check.checkToggleSwitch(
+			key_productName = "Torque Converters",
+			locator1 = "CommerceEntry#SELECT_CHECKBOX");
+
+		for (var key_value : list "1,2,3") {
+			AssertElementPresent(locator1 = "CommerceEntry#ASSERT_MINI_COMPARE");
```

Comment:
As for the comment above, please update `CommerceEntry#ASSERT_MINI_COMPARE` locator name.